specs/deposits.md

```diff
@@ -44,41 +69,36 @@ Picking a high identifier minimizes the risk that the identifier will be used be
 transaction type on the L1 chain in the future. We don't pick `0x7F` itself in case it becomes used
 for a variable-length encoding scheme.
 
-### Uses of the Deposit Transaction Type
+### Kinds of Deposited Transactions
 
-Although in practice we define only one new transaction type we can distinguish between two distinct
-situations which occur in the deposit block, based on their positioning.
+Although we define only one new transaction type, we can distinguish between two kinds of deposited
+transactions, based on their positioning in the [deposit block][g-deposit-block]:
 
-1. The first transaction MUST be a [L1 attributes deposit][l1-attributes-deposit], followed by
-2. an array of zero-or-more [L1 transaction deposits][l1-transaction-deposits] submitted to the
-deposit feed contract on L1.
+1. The first transaction MUST be a [L1 attributes deposited transaction][l1-attr-deposit], followed
+   by
+2. an array of zero-or-more [user-deposited transactions][user-deposited] submitted to the deposit
+   feed contract on L1.
 
-The rationale for creating only one new transaction type is to minimize both
-modifications to L1 client software and complexity in general.
+We only define a single new transaction type in order to minimize modifications to L1 client
+software, and complexity in general.
 
-> **TODO** Specify and link to deposit blocks
```

Comment:
This is defined in the rollup node, and already implicitly in this file (first tx is l1 attributes, then user-deposited), so I don't think it's necessary to add something here.

Also we link to the glossary.